jobs/integr8ly/ocp3/test-suites/3scale-test.yaml

```diff
@@ -0,0 +1,80 @@
+---
+
+- job:
+    name: 3scale-tests
```

Comment:
the name indicates running number of tests. Perhaps rename it to `3scale-test`?

Reply:
OK